README.md

```diff
@@ -27,6 +28,46 @@ To install the SDK, run the following command using pip or a package manager of
 pip install harambe-sdk
 ```
 
+## SDK Methods
+
+### sdk.capture_html()
+Captures the raw HTML of the `selector` passed and defaults to capturing the entire document.
+If any `exclude_selectors` are passed, they are excluded from the capture.
+The captured HTML is saved to the server and the URL is returned.
```

Comment:
would need to explain that on platform we actually download the file on your behalf.

A code example would also be useful here